talent/contract.md

```diff
@@ -1,19 +1,25 @@
 4. The contract hs an indefinite duration and the Spanish standard of 6 trial months. <b>This means that...</b>
 5. This clause details how much your gross salary will be.
-6. At source{d}, we offer 23 day of holidays a year. We have a <a href="">holiday policy</a> by which you can work on bank holidays or weekends though and exchange it for more days. Also, any unused holidays will be carried over to the next year. <b>link missing</b>
-7. x
-8. The legislation of the statute of workers approved by the <a href="https://www.boe.es/buscar/doc.php?id=BOE-A-1995-7730">Royal Legislative Decree 1/1995 of 24th March</a> and the collective agreement of <a href="https://www.boe.es/boe/dias/2009/04/04/pdfs/BOE-A-2009-5688.pdf"><k>Empresas de Consultoría y Estudios Opinión Pública</k></a> will be applied to this contract.
+6. At source{d}, we offer 23 day of holidays a year. We have a holiday policy by which you can work on bank holidays or weekends though and exchange it for more days. Also, any unused holidays will be carried over to the next year. Also, we have an additional clause including more details.
+7. The legislation of the statute of workers approved by the <a href="https://www.boe.es/buscar/doc.php?id=BOE-A-1995-7730">Royal Legislative Decree 1/1995 of 24th March</a> and the collective agreement of <a href="https://www.boe.es/boe/dias/2009/04/04/pdfs/BOE-A-2009-5688.pdf"><k>Empresas de Consultoría y Estudios Opinión Pública</k></a> will be applied to this contract.
+8. x
 9. x
-10. Just stating that the public authorities will be notified of this work contract in the next 10 days.<br>
-### Additional clauses:<br>
+10. Just stating that the public authorities will be notified of this work contract in the next 10 days.
+
+
+### Additional clauses:
 1. All IP developed during work at source{d} will be owned by source{d} entirely.
 2. The confidentiality clause frobids anyone from sharing confidential information unless it has been otherwise instructed by source{d} or unless it is in the benefit of source{d}. Regardless of this we're very open about everything we do as you can see in our <a href="https://blog.sourced.tech/">engineering blog</a>, <a href="https://github.com/src-d">open-source projects</a>, <a href="https://github.com/src-d/tutorial">source{d} guide</a> or <a href="https://medium.com/source-d">company blog</a>.
 3. This clause is about how source{d} will handle your data according to Spanish Data Protection Law.
+4. Prorated Extraordinary Payments, this means we make 12 payments per year. 
+5. [Remote work](https://github.com/src-d/guide/blob/master/talent/flexible_holidays_working_schedule_remote_work.md)
+6. [Holidays](https://github.com/src-d/guide/blob/master/talent/flexible_holidays_working_schedule_remote_work.md)
```

Comment:
link would be better if it would be:
`[holiday-policy](flexible_holidays_working_schedule_remote_work.md#holiday-policy)`